problem-26.c

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
+// Stores “Hello World” to the text file and then shows content of the file on terminal after reading
+
+int main()
+{
+    FILE *file;
+
+    file = fopen("text-files/hello.txt", "w");

```

Comment:
Do we need to have `hello.txt` file in codebase or can we ignore it via `.gitignore`?

Reply:
Same feedback for all files.